static/css/zamboni/reviewers.less

```diff
@@ -1129,9 +1129,33 @@ pre.history-comment {
     font-style: italic;
 }
 
-.history-notes {
-    max-width: 370px;
```

Comment:
I'm assuming this removal was deliberate, but noting because it's width and the PR is about height

Reply:
It's to be consistent between the way we display "approval notes" and comments